### PR TITLE
Leader gear rack ammunitions

### PR DESCRIPTION
**Pull request by z9ctn4fn**
#936

### CODE REVIEW

**Comment by DrSmugleaf** [comment]
Thanks for the PR, though next time do not merge a different open PR into your own, it will make every file changed conflict as we squash before merging. If you need to make changes to an open PR, open a PR to their branch or use the "Suggest changes" system in PR review.

**Review by DrSmugleaf — APPROVED** [review]

**Review by DrSmugleaf — APPROVED** [review]

**Comment by z9ctn4fn** [comment]
oh thanks

**Pull request merged**
